modules/osdk-aws-sts-enabling.adoc

```diff
@@ -0,0 +1,319 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-token-auth.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-aws-sts-enabling_{context}"]
+= Enabling Operators to support CCO-based workflows with AWS STS
+
+As an Operator author designing your project to run on Operator Lifecycle Manager (OLM), you can enable your Operator to authenticate against AWS on STS-enabled {product-title} clusters by customizing your project to support the Cloud Credential Operator (CCO).
+
+With this method, the Operator is responsible for creating the `CredentialsRequest` object, which means the Operator requires RBAC permission to create these objects. Then, the Operator must be able to read the resulting `Secret` object.
+
+[NOTE]
+====
+Pods related to the Operator deployment mount a `serviceAccountToken` volume for the secret by default.
+====
+
+.Prerequisities
+
+* {product-title} 4.14 or later
+* Cluster in STS mode
+* OLM-based Operator project
+
+.Procedure
+
+. Update your Operator project's `ClusterServiceVersion` (CSV) object:
+
+.. Ensure your Operator has RBAC permission to create `CredentialsRequests` objects:
++
+.Example `clusterPermissions` list
+[%collapsible]
+====
+[source,yaml]
+----
+# ...
+install:
+   spec:
+     clusterPermissions:
+     - rules:
+       - apiGroups:
+         - "cloudcredential.openshift.io"
+         resources:
+         - credentialsrequests
+         verbs:
+         - create
+         - delete
+         - get
+         - list
+         - patch
+         - update
+         - watch
+----
+====
+
+.. Add the following annotation to claim support for STS:
```

Comment:
this label more specifically claims support for this specific way of doing STS (by creating a credReq etc.). I think it would be good to clarify this since there are many ways to support STS but since this label changes to UI and install flow, it should only be used in this specific case of STS support.

Reply:
Changed to:
```
.. Add the following annotation to claim support for this method of CCO-based workflow with AWS STS:
```
LMK if we should get even more specific than that short-hand. For example, could try to quickly restate this part of this module's intro:
> With this method, the Operator is responsible for creating the `CredentialsRequest` object, which means the Operator requires RBAC permission to create these objects.